docs/sources/k6/next/examples/http-authentication.md

```diff
@@ -45,44 +45,13 @@ export default function () {
     },
   };
 
-  res = http.get(`https://httpbin.test.k6.io/basic-auth/${username}/${password}`, options);
-
-  // Verify response (checking the echoed data from the httpbin.test.k6.io
-  // basic auth test API endpoint)
-  check(res, {
-    'status is 200': (r) => r.status === 200,
-    'is authenticated': (r) => r.json().authenticated === true,
-    'is correct user': (r) => r.json().user === username,
-  });
-}
-```
-
-{{< /code >}}
-
-## Digest authentication
```

Comment:
Digest auth is deprecated since 2011 (https://en.wikipedia.org/wiki/Digest_access_authentication) so I would not recommend users to implement it in their HTTP servers. I also think this auth method is not widely used nowadays - for this reason I've removed this section.